pootle/apps/pootle_translationproject/views.py

```diff
@@ -25,7 +25,7 @@
 from staticpages.models import StaticPage
 
 
-ANN_COOKIE_NAME = 'project-announcements'
+ANN_COOKIE_NAME = 'sidebar-cookie'
```

Comment:
And at the same time `ANN_COOKIE_NAME` should be renamed as well; `SIDEBAR_COOKIE_NAME` or similar.